src/ptvsd/_vendored/pydevd/_pydevd_bundle/_debug_adapter/pydevd_custom_schema.py

```diff
@@ -0,0 +1,180 @@
+from .pydevd_base_schema import BaseSchema, register, register_request, register_response
```

Comment:
The schema file is currently generated by `_pydevd_bundle._debug_adapter.__main__pydevd_gen_debug_adapter_protocol` and I think that instead of manually coding the Python custom schema we should create a `debugProtocolCustom.json` and in `gen_debugger_protocol` load both files creating a dict where the `definitions` contains contents of both (if the generator changes to create a different structure in the future we shouldn't have to manually update a separate file).